x-pack/plugins/apm/server/lib/services/get_service_dependencies/get_metrics.ts

```diff
@@ -65,8 +65,12 @@ export const getMetrics = async ({
               timeseries: {
                 date_histogram: {
                   field: '@timestamp',
-                  fixed_interval: getBucketSize({ start, end, numBuckets })
-                    .intervalString,
+                  fixed_interval: getBucketSize({
+                    start,
+                    end,
+                    numBuckets,
+                    searchAggregatedTransactions: true,
```

Comment:
This naming is a little off. It's got nothing to do with aggregated transactions. Can we find a better name?

Reply:
It's called `minInterval` in Observability:
https://github.com/elastic/kibana/blob/61d23665a9e391c3cfa42021671ef3461f0c9560/x-pack/plugins/observability/public/utils/get_bucket_size/index.ts#L16

```suggestion
                    minInterval: true,
```

Reply:
Probably `minBucketSize` is better than `minInterval`:

```suggestion
                    minBucketSize: P0,
```